cypress/support/index.d.ts

```diff
@@ -2,7 +2,10 @@
 
 declare namespace Cypress {
   interface Chainable<Subject> {
-    getByDataCy(selector: string): Chainable<any>
-    register(email: string, username: string, password: string): Chainable<any>
+    getByDataQa(selector: string): Chainable<any>;
```

Comment:
The getByDataCy method seems to be duplicated which is unnecessary and may cause confusion in the future. Consider deleting one instance.